### ISSUE TITLE
Impossible to see Add VPN inputs

### ISSUE THREAD

**Issue by lelledaniele** [theinvisible/openfortigui#11]
Hi,

is impossible to understand/edit the inputs on my Galago Pro. 3200x1800 resolution.
I tried to resize the `Add VPN` window but it was impossible.
I tried to change the screen resolution but I didn't work. So, I am not 100% sure what is causing this issue.
I am using Ubuntu (Unity) 17.04.

![screenshot from 2017-08-30 21-25-41](https://user-images.githubusercontent.com/6173422/29893621-93c029a0-8dca-11e7-8f2c-ce1628d0c28a.png)

Many thanks,
Lelle

**Comment by theinvisible** [comment]
Hello,

are you using non-default font sizes? 

The problem here i think is that the dialog windows are fixed size and the whole layout is too big for the dialog window size. For a quick fix i will make the dialog windows resizeable. Should be in the following git updates, will update this issue here.

**Comment by lelledaniele** [comment]
Hi @theinvisible,

I have seen your [commit](https://github.com/theinvisible/openfortigui/commit/6d3f85f72fa4277718fdc9f88955f4af0af851f3) to resize the dialog box.
I have followed [this instuctions](https://hadler.me/linux/openfortigui/#archives-2) to build from GIT repository. But I am stilling have the `Add VPN` dialog box not resizable.
I did not change the font size. Also, I have tried to restore the Ubuntu default theme. I have attached the fonts settings. I am now with Ubuntu GNOME 17.04.

![screenshot from 2017-09-24 08-32-27](https://user-images.githubusercontent.com/6173422/30780463-1a86be86-a103-11e7-84e1-e6e0c546a3a0.png)

Many thanks,
Lelle
